pom.xml

```diff
@@ -247,27 +254,45 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>io.cdap.cdap</groupId>
-      <artifactId>cdap-data-pipeline</artifactId>
-      <version>${cdap.version}</version>
-      <scope>test</scope>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-mapreduce-client-core</artifactId>
+      <version>${hadoop.version}</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>io.cdap.cdap</groupId>
-      <artifactId>cdap-test</artifactId>
-      <version>${cdap.version}</version>
+      <groupId>io.cdap.plugin</groupId>
+      <artifactId>format-avro</artifactId>
+      <version>${hydrator.version}</version>
     </dependency>
     <dependency>
-      <groupId>io.cdap.cdap</groupId>
-      <artifactId>cdap-unit-test</artifactId>
-      <version>${cdap.version}</version>
-      <scope>test</scope>
-      <exclusions>
-        <exclusion>
-          <groupId>asm</groupId>
-          <artifactId>asm</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>io.cdap.plugin</groupId>
+      <artifactId>format-blob</artifactId>
```

Comment:
why do we need all of them?

Reply:
https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L181

Reply:
My initial thought was to have all the formats in _ETLBatchTestBase.java_ same as we have them in **hydrator**. 

https://github.com/data-integrations/ftp-plugins/blob/c40deec6eea51681e1c6ab1c4812dd0f29420e32/src/test/java/io/cdap/plugin/batch/ETLBatchTestBase.java#L97-L121